README.md

```diff
@@ -138,7 +138,7 @@ echo file_get_contents('path/to/package/icons/simpleicons.svg');
 ?>
 ```
 
-## Third Party Extensions
+## Third-Party Extensions
```

Comment:
I don't think this hyphen needs adding, though there are uses of both variations across the internet. Thoughts, @simple-icons/maintainers?

Reply:
I believe it's meant to be hyphenated when used as an adjective, but can be with a space when used as a noun.

So grammatically speaking, this usage would be better with a hyphen. If you'd rather I not change it, I can put it back though.

Reply:
Figured it'd be something like that, though the use cases of the phrase I find seem to mix and match between the two, hence why I've pinged the other maintainers to gain their thoughts on it. If there's no objections by the end of the day, I'll look to merge this one in.

Reply:
I can see a lot of references about hyphenation of compound words when used as adjetves. Altough I'm not a native english speaker, this change seems good to me.

Reply:
Marking this as resolved, and merging this PR due to multiple other maintainers approving. Thanks for your work @DenverCoder1